tensorflow/lite/tools/pip_package/build_pip_package_with_bazel.sh

```diff
@@ -88,7 +88,7 @@ case "${TENSORFLOW_TARGET}" in
     ;;
 esac
 
-bazel build -c opt -s --config=monolithic --config=noaws --config=nogcp --config=nohdfs --config=nonccl \
+bazel ${BAZEL_ARGS} build -c opt -s --config=monolithic --config=noaws --config=nogcp --config=nohdfs --config=nonccl \
```

Comment:
How about ${BAZEL_STARTUP_OPTIONS}? It looks clearer.

Reply:
Yes I it looks clearer, I  will push an update